### PR TITLE
[bitnami/postgresql] feat: :sparkles: :lock: Add readOnlyRootFilesystem support 

### PR DESCRIPTION
**Pull request by javsalgar**
Signed-off-by: Javier Salmeron Garcia <jsalmeron@vmware.com>

### Description of the change

This PR performs the necessary changes to enable readOnlyRootFilesystem. This consists of:

- Adding emptyDir directories in all container writable paths
- Update postgresql image with one compatible with an emptydir conf folder
- Add runAsGroup

IMPORTANT: In order to avoid potentially breaking changes, this PR sets the readOnlyRootFilesystem value to `false`.

### Benefits

More security in the chart

### Possible drawbacks

Potential unforeseen issues with customizations

### Applicable issues

- fixes #

### Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [x] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami/readme-generator-for-helm)
- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/postgresql/templates/primary/statefulset.yaml [hunk 1]
```diff
@@ -156,13 +158,13 @@ spec:
           securityContext: {{- .Values.volumePermissions.containerSecurityContext | toYaml | nindent 12 }}
           {{- end }}
           volumeMounts:
-            {{- if .Values.primary.persistence.enabled }}
```
Comment (migruiz4) [inline bitnami/postgresql/templates/primary/statefulset.yaml:159]: You did not apply this same change in the `secondary` node statefulset.

**Review by migruiz4 — COMMENTED** [review]
Please take a look at my comment, you introduced differences between primary and secondary nodes handling `persistence.enabled`.

**Review by migruiz4 — APPROVED** [review]
LGTM!

**Comment by jl-beast** [comment]
this is very cool. thanks for the hard work. I appreciate this.

**Pull request merged**